### PR TITLE
notifi-dapp-example (ODF specific): make sure localStorage is removed before route back to root

### PR DESCRIPTION
**Pull request by eric-notifi**
- [x] Describe the purpose of the change
As title, Try to resolve the edge case the the infinite spin showing up after re-deploying

- [ ] Create test cases for your changes if needed
No need
- [ ] Include the ticket id if possible (Collaborator only)
N/A

### CODE REVIEW

File: packages/notifi-dapp-example/src/hooks/useNotifiRouter.ts [hunk 1]
```diff
@@ -53,12 +53,16 @@ export const useNotifiRouter = () => {
...
       if (selectedWallet) {
         wallets[selectedWallet].disconnect();
       } else if (injectiveSelectedWallet) {
         injectiveWallets[injectiveSelectedWallet].disconnect();
       }
+      setTimeout(() => {
+        /** NOTE: preserve 0.5s to make sure cache is cleaned before route back to root dir*/
+        console.log(`LOGIN ERROR: ${JSON.stringify(loginError)}`);
+        handleRoute('/');
+      }, 5000);
```
Comment (ankush-notifi) [inline packages/notifi-dapp-example/src/hooks/useNotifiRouter.ts:65]: I noticed that the note mentions a 0.5-second delay to ensure the cache is cleaned before routing back to the root directory. However, the code currently has a 5-second delay. Could you please confirm if this is intentional or if it should be 500 milliseconds instead of 5000 milliseconds as per the note?

cc @eric-notifi

Comment (eric-notifi) [reply packages/notifi-dapp-example/src/hooks/useNotifiRouter.ts:65]: Nice catch!! I was gona do 500. Thanks @ankush-notifi.
